features/fixtures/shared/scenarios/NSExceptionShiftScenario.m

```diff
@@ -1,4 +1,5 @@
 #import "Scenario.h"
+#import "Logging.h"
```

Comment:
Do we really need to include the header in all the scenario files, when there are no other changes?

Reply:
Scenario.h is currently the bridging header, so anything it sees, Swift sees, which causes problems with the naming clash. We could try hoisting the things Swift needs out into another header file, but I ran out of time. I can look again later.